pool/src/lib.rs

```diff
@@ -793,37 +844,79 @@ impl LendingPool {
             })?
     }
 
-    /// Returns collateral_accrued_rate corrected for current time
-    fn get_collateral_coeff(env: &Env, reserve: &ReserveData) -> Result<FixedI128, Error> {
+    /// Returns lender accrued rate corrected for the current time
+    fn get_actual_lender_accrued_rate(
+        env: &Env,
+        reserve: &ReserveData,
+    ) -> Result<FixedI128, Error> {
         let current_time = env.ledger().timestamp();
         let elapsed_time = current_time
             .checked_sub(reserve.last_update_timestamp)
             .ok_or(Error::CollateralCoeffMathError)?;
-        let prev_ar = FixedI128::from_inner(reserve.collat_accrued_rate);
+        let prev_ar = FixedI128::from_inner(reserve.lender_accrued_rate);
         if elapsed_time == 0 {
             Ok(prev_ar)
         } else {
             let lend_ir = FixedI128::from_inner(reserve.lend_ir);
-            calc_accrued_rate_coeff(prev_ar, lend_ir, elapsed_time)
+            calc_next_accrued_rate(prev_ar, lend_ir, elapsed_time)
                 .ok_or(Error::CollateralCoeffMathError)
         }
     }
 
-    /// Returns debt_accrued_rate corrected for current time
-    fn get_debt_coeff(env: &Env, reserve: &ReserveData) -> Result<FixedI128, Error> {
+    /// Returns collateral coefficient
+    /// collateral_coeff = [underlying_balance + collat_accrued_rate * total_debt_token]/total_stoken
```

Comment:
Let's replace the mention of the `collat_accrued_rate` to the `lender_accrued_rate`.